README.md

```diff
@@ -13,7 +13,15 @@ Start server at port 8000
 ```
 $ make watch
 ```
-Run the tests
+### Create superuser to administer the page:
 ```
-$ make test
+$ python manage.py createsuerpuser
```

Comment:
The p is at the wrong place. :)
And adding `source venv/bin/activate` would also be good!

Reply:
Ups, sorry ...fixed it:)